GETTING_STARTED.md

```diff
@@ -72,7 +72,19 @@ apply(plugin = "godot-gradle-plugin")
 repositories {
     mavenLocal()
     maven("https://dl.bintray.com/utopia-rise/kotlin-godot")
-    jcenter()
+    maven(url = "https://dl.bintray.com/utopia-rise/kotlinx")
+    jcenter {
+        content {
+            excludeModule("org.jetbrains.kotlinx", "kotlinx-coroutines-core-native")
+            excludeModule("org.jetbrains.kotlinx", "atomicfu-native")
+        }
+    }
+    mavenCentral {
+        content {
+            excludeModule("org.jetbrains.kotlinx", "kotlinx-coroutines-core-native")
+            excludeModule("org.jetbrains.kotlinx", "atomicfu-native")
+        }
+    }
```

Comment:
I would add a comment directly in here to state for what the change is for and why the user needs to use our repo for coroutines support rather than the jetbrains one. This is not clear if the user just looks at this file. To me without prior knowlegde this would look fishy...